community-concepts/productaanvraag/productaanvraag-dimpact.json

```diff
@@ -302,19 +302,19 @@
       "betrokkenen" : [
         {
           "inpBsn" : "111222333",
-          "roltypeOmschrijving" : "initiator",
```

Comment:
Dit nieuwe  `roltypeOmschrijving ` moet niet verwijderd worden toch @andyverberne ? Die hebben wij in ZAC juist nodig voor: https://dimpact.atlassian.net/browse/PZ-3471 Ik dacht dat het idee was dat we beide velden in de spec wilden hebben?

Reply:
Aja, dit gaat om de voorbeelden. Prima zo :-)